charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/helm-addon-installation.yaml

```diff
@@ -27,4 +27,11 @@ data:
         - effect: NoSchedule
           key: node-role.kubernetes.io/control-plane
     ### <NFD-WORKER-CONF-END-DO-NOT-REMOVE>
+
+    gc:
+      tolerations:
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/master
```

Comment:
`node-role.kubernetes.io/master` is deprecated. we should not expect it on the CAREN created cluster.

Reply:
While that is true, we have 38 uses of that in CAREN. Let's replace it in a separate PR, please.